### ISSUE TITLE
Config error?

### ISSUE THREAD

**Issue by swin66** [Chouffy/home_assistant_tgtg#53]
I've upgraded to the latest version and rerun the script for my config and pasted it in, but now my ha won't restart.

The system cannot restart because the configuration is not valid: Invalid config for [sensor.tgtg]: [email] is an invalid option for [sensor.tgtg]. Check: sensor.tgtg->email. (See ?, line ?).

**Comment by Chouffy** [comment]
Hi @swin66,
Home Assistant checks your `configuration.yaml` _before_ restarting HA. With the latest update, we changed one of the config key from `username`  to `email`. 
If you update your `configuration.yaml` file while HA is running, then HA will see the new `email` key but won't have it in the memory as the `tgtg` module will be updated _after_ reboot (if you updated by HACS).

TL;DR: you need to stop and start Home Assistant manually, outside of HA itself :)
